### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by ibugithub**
- In this branch :
- We added a popup window to show the projects.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,23 +235,18 @@ <h2>Multi-Post Stories</h2>
...
-                        <button class="btn">See project</button>
-                    </a>
+                        <button id="project3" class="btn work">See project</button>
                 </div>
             </div>
 
         </div>
 
-        <div class="work3">
+        <div id="project3" class="work3">
```
Comment (Ibizugbe) [inline index.html:244]: Nice job adding the popup functionality👍 Kindly note that you are required in this milestone to render the popup and project section dynamically from the array of objects you have created that contains the details about each project😉 do well to remove the project section from your `html`  file and render it dynamically with `js`👍.

generate popup and projects section dynamically

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @ibugithub 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] The popup functionalities works properly👍.


## Required Changes ♻️

- [ ] Kindly ensure your popup matches the design both on mobile and on desktop😉 I suggest you make comparison with the design and make the changes where necessary as we are required to make the popup responsive and stick to the design in this milestone😉😉

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL]  Your commit messages are descriptive😉 but let's ensure they are capitalised in subsequent milestones as this is a good practice for writing professional commit messages👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ibugithub) [reply index.html:244]: @Ibizugbe  Still a little confused about what I'll have to change here. Would you please explain more?

Comment (Ibizugbe) [reply index.html:244]: From the requirements of [this milestone](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md), the projects or work section of your page shouold be rendered dynamically. currently, you still have your project section in your `.html` file. The instruction from the milestone requires you to take away your project section from the `.`html` file and generating it using javascript👍.

**Review by andersonlebon — APPROVED** [review]
## Hi @ibugithub 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**